### ISSUE TITLE
Missing support for VS 2015

### ISSUE THREAD

**Issue by Saibamen** [ilammy/msvc-dev-cmd#7]
In ROS we need to build with VS 2015: https://github.com/reactos/reactos/pull/2688

Extravert-ir created fork: please see his changes at: https://github.com/Extravert-ir/msvc-dev-cmd/commits/release/v1

**Comment by ilammy** [comment]
Hey, @Saibamen, @Extravert-ir! I'm sorry for taking so long to respond.

I've gave a shot an integrating changes by @Extravert-ir in #9, and published a new version 1.3.0. Could you please try it out?

```
      - name: Enable Developer Command Prompt
        uses: ilammy/msvc-dev-cmd@v1.3.0
```

**Comment by ilammy** [comment]
Well, uh... To the best of my ability, I believe that this issue should be resolved now.

Please reopen if it does not work for you.
